File: app/src/main/java/can/siempredelao/f1kotlin/backend/MRDataManufacturers.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

public class MRDataManufacturers {

    public String           xmlns;
    public String           series;
    public String           url;
    public String           limit;
    public String           offset;
    public String           total;
    @SerializedName("ConstructorTable")
    public ConstructorTable constructorTable;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/MRDataManufacturers.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

/** Collection of manufacturers */
class MRDataManufacturers(val xmlns: String,
                          val series: String,
                          val url: String,
                          val limit: String,
                          val offset: String,
                          val total: String,
                          @SerializedName("ConstructorTable") val constructorTable: ConstructorTable)
